src/jdk.management.agent/share/classes/sun/management/jmxremote/ConnectorBootstrap.java

```diff
@@ -297,15 +297,15 @@ public static void unexportRegistry() {
      /**
       * Initializes and starts the JMX Connector Server.
       * If the com.sun.management.jmxremote.port property is not defined,
-      * simply return. Otherwise, attempts to load the config file, and
+      * simply returns. Otherwise, attempts to load the config file, and
       * then calls {@link #startRemoteConnectorServer(java.lang.String,
       * java.util.Properties)}.
       *
       * This method is used by some jtreg tests.
       **/
       public static synchronized JMXConnectorServer initialize() {
 
-         // Load a new management properties
+         // Load new management properties
```

Comment:
Debatable. I guess what was meant here is:
```
// Loads a new management Properties instance
```
Otherwise LGTM.

Reply:
I've reverted that particular line.